packages/bpk-component-table/src/BpkTableCell-test.js

```diff
@@ -22,7 +22,7 @@ import BpkTableCell from './BpkTableCell';
 
 describe('BpkTableCell', () => {
   it('should render correctly', () => {
-    const tree = renderer.create(<BpkTableCell>Tabular data</BpkTableCell>).toJSON();
+    const tree = renderer.create(<BpkTableCell className="test">Tabular data</BpkTableCell>).toJSON();
```

Comment:
Can you add an additional test case (`it(...)`) for this configuration? See for example https://github.com/Skyscanner/backpack/blob/master/packages/bpk-component-table/src/BpkTable-test.js#L33